scripts/util.py

```diff
@@ -51,8 +52,17 @@ def read_source_root_path():
 def read_proof_root_path():
     return read_path_from_stdin("the 'proofs' directory (usually '.')")
 
+def is_litani_installed():
+    dialogue = textwrap.fill("Litani can either be distributed with your " \
+        "repository in the form of a submodule, in which case you will need " \
+        "to provide the desired directory, or it can be installed using: " \
+        "brew install aws/tap/litani")
+    print(dialogue)
+    print("Has litani already been installed in your system? [Y/n]", end="")
+    return read_from_stdin().lower() in ("y", "yes")
+
```

Comment:
Will this be better?
```
def is_litani_installed():
    dialogue = textwrap.fill("Litani can either be distributed with your " \
        "repository in the form of a submodule, in which case you will need " \
        "to provide the desired directory, or it can be installed using: " \
        "brew install aws/tap/litani")
    print(dialogue)
    if shutil.which("litani"):
        LITANI_VERSION = subprocess.run(
            "litani -V",
            shell=True,
            capture_output=True).stdout.decode("utf-8").rstrip()
        print(f"litani-{LITANI_VERSION} Installation found")
    print("Do you wanna use this installation of Litani? [Y/n]", end="")
    return read_from_stdin().lower() in ("y", "yes")
```

Reply:
I incorporated Ronak's suggestion with a small change of my own